### ISSUE TITLE
The Local label is being written over by `deck init` command if present.

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#72]
While I was making test described in the issue #71 I've discovered an unexpected behavior of the `deck init` command:

- The deck `a2459e054ce0f600c90be458915af6bad36a6863a0ce0e33ab76086b514f765a` that was originally labeled as `DTNewSpecv2_newl` after I've run  `deck init a2459e054ce0f600c90be458915af6bad36a6863a0ce0e33ab76086b514f765a` has changed it's local label into `DTNewSpecv2` (the global name of the deck).
- When I've tried to modify the local label using `deck set DTNewSpecv2_newl DTNewSpecv2 -m`  command I've got `Error: Value already exists, you can't change it in protected mode.`
- After I've deleted the `DTNewSpecv2` using the `deck set DTNewSpecv2 -d -n` command the output of the ` deck list -p` has become:
```
| Local label      | ID                                                               | Global name               | Issuer                             | M | Conf.  | I |
+------------------+------------------------------------------------------------------+---------------------------+------------------------------------+---+--------+---+
| DTNewSpecv2_newl | a2459e054ce0f600c90be458915af6bad36a6863a0ce0e33ab76086b514f765a | DTNewSpecv2               | mvrm2HAoKqiCmeQiwKMuEFpeEn7rJEmpMz | 1 | 270179 | + |
|                  | 3369f3eb703656c0b46b613dcfb4df78fc05657a0b75f44551634de6321d9d9c | dt_short                  | mwzox84gMy8TKdeBftrfpmwGVZnJJdaReg | 1 | 263638 | + |
|                  | 0b3816628374d24078c741251f712b7f6af1c6f82ff7ddce083c20a58fcc2bc1 | impossible_donation_token | mpmZCVusQ6WoWHcFTAizmEcqSXdu5YX5PV | 1 | 31669  |   |
```
Which makes me think that the original Local label were stored somewhere after it was "covered" by the `deck init` command's label.

**Comment by d5000** [comment]
I can confirm this bug. Thanks for reporting.

The reason was actually that the  `deck init` command didn't overwrite the old label, but instead created a second label.

However this label won't be shown in `deck list` because this command lists the decks not by stored label, but by deck ID, and then adds the first label it finds for it.

I added a check to avoid `deck init` storing the label if it finds an existing one. I'll check if there are other implications of this problem, maybe I'll change the way `deck list` works in the case more than one (local) label were stored for it (this is not included in the commit).

Additionally, I made a small CLI change to make terminology consistent: `deck init -s` was renamed to `deck init --cache` or `-c`.

Commit: 18aa4d7

**Comment by buhtignew** [comment]
This works.

> Additionally, I made a small CLI change to make terminology consistent: deck init -s was renamed to deck init --cache or -c.

I think we should change `pacli deck init [DECK] -s [BLOCKS] [-a]` into `pacli deck init [DECK] -c [BLOCKS] [-a]` then in the help usage modes section.

**Comment by d5000** [comment]
Of course, done. Will be uploaded in the next commit with actual code changes.

**Comment by d5000** [comment]
Uploaded commit (775cd18) & closed.
